netci.groovy

```diff
@@ -725,8 +726,15 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
             assert scenario == 'default'
             switch (os) {
                 case 'Ubuntu':
-                    Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} Cross ${configuration} Build", "(?i).*test\\W+${os}\\W+${architecture}.*")
-                    break
+                    switch(architecture) {
+                        case "arm":
+                            // Removing the regex will cause this to run on each PR.
+                            Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} Cross ${configuration} Build")
```

Comment:
Something changed from this morning. I have a clean build as of 5 minutes ago on the vm and hardware.